src/us/kbase/userandjobstate/jobstate/JobState.java

```diff
@@ -579,12 +635,6 @@ public void deleteJob(final String user, final String jobID,
 			throw err;
 		}
 		
-		final DBObject query = new BasicDBObject(MONGO_ID, id);
```

Comment:
Why were these here before and why are they no longer needed?

Reply:
This was written in 2013 so I'm not totally sure why I did things this way, but the `querystr` variable and the `query` object wind up being the exact same query under the hood. Since I'm not using `querystr`, I can just reuse the `query` object that replaces it in both places.

Reply:
I guess I used jongo for the `findOne` query since it maps the result to a `Job`, and used the standard syntax for `remove` since I didn't need the mapping.